src/Sentry/Laravel/Features/QueueIntegration.php

```diff
@@ -181,12 +180,22 @@ public function handleJobProcessingQueueEvent(JobProcessing $event): void
 
         $resolvedJobName = $event->job->resolveName();
 
+        $receiveLatency = null;
+        if ($event->job->payload()[self::QUEUE_PAYLOAD_PUBLISH_TIME] !== null) {
+            $receiveLatency = microtime(true) - $event->job->payload()[self::QUEUE_PAYLOAD_PUBLISH_TIME];
+        }
+
         $job = [
-            'job' => $event->job->getName(),
-            'queue' => $event->job->getQueue(),
-            'resolved' => $resolvedJobName,
-            'attempts' => $event->job->attempts(),
-            'connection' => $event->connectionName,
+            'messaging.system' => 'laravel',
+            
+            'messaging.destination.name' => $event->job->getQueue(),
+            'messaging.destination.connection' => $event->connectionName,
+            
+            'messaging.message.id' => (string) $event->job->getJobId(),
+            'messaging.message.envelope.size' => mb_strlen($event->job->getRawBody()),
+            'messaging.message.body.size' => mb_strlen(json_encode($event->job->payload()['data'])),
```

Comment:
You want to count bytes here, `mb_*` will count multibyte characters as a single byte instead. `strlen` can be used to count bytes ([see](https://www.php.net/manual/en/function.strlen.php#112411)).

```suggestion
            'messaging.message.envelope.size' => strlen($event->job->getRawBody()),
            'messaging.message.body.size' => strlen(json_encode($event->job->payload()['data'])),
```